nclib/netcat.py

```diff
@@ -215,35 +216,39 @@ def _parse_target(target, listen, udp, ipv6):
             if target.startswith('nc '):
                 out_host = None
                 out_port = None
-                pieces = target.split()[1:]
-                while pieces and pieces[0][0] == '-':
-                    if pieces[0] == '-u':
+
+                try:
+                    opts, pieces = getopt.getopt(target.split()[1:], 'tu46lp:',
```

Comment:
The original set of options I added were options that nc(1) can take, the point being that you can give nclib.Netcat a nc(1) command line as input. My version of netcat has this:

```
     -t      Causes nc to send RFC 854 DON'T and WON'T responses to RFC 854 DO
             and WILL requests.  This makes it possible to use nc to script
             telnet sessions.
```

so we probably don't want that, or -4

Reply:
In my version (netcat (The GNU Netcat) 0.7.1):

> -t
> --tcp
> Selects the TCP protocol, this is the default.  It may be useful (see Tunnel Mode) to specify this option after for example the UDP option in order to allow a cross-protocol bridge between TCP and UDP.

But I do agree I shouldn't add new features in a *refactoring* pull request.